server/templates/EditChart.template.js

```diff
@@ -0,0 +1,28 @@
+import serialize from 'serialize-javascript'
+
+import basePathConfig from '../configs/basePathConfig'
+
+const basePath = basePathConfig || ''
```

Comment:
should we default to an empty string in `basePathConfig` so we don't need to define this variable?

Reply:
This part is tricky because this file and the path is written by a script 
https://github.com/AMP-SCZ/dpdash/blob/7c827e1e21d1f01a50d03a1b7b04eeece7482de6/singularity/configure.py#L414

And the directory where the file lives is not committable 
https://github.com/AMP-SCZ/dpdash/blob/7c827e1e21d1f01a50d03a1b7b04eeece7482de6/.gitignore#L15

My concern by defaulting to just an empty string is that there could be an actual path in production. We could add a card to the project to look into it some more. I think we would need to know more from MGH team.